style/row.less

```diff
@@ -1,4 +1,6 @@
 .rdg-row {
+  contain: strict;
```

Comment:
Do we need to set contain at the row level? Does it result in better performance? Or is it mainly to fix shadows? Should change the shadow css instead?

Reply:
It should result in better performance as the browser has more opportunity to optimize rendering. We can safely enable it afaict, so I think it's worth keeping.